### PR TITLE
METS: Handle empty rows in metadata.csv

### PR DESCRIPTION
**Pull request by Hwesta**
Skip empty rows when parsing a metadata.csv

### CODE REVIEW

File: src/MCPClient/tests/test_create_aip_mets.py [hunk 1]
```diff
@@ -348,3 +348,20 @@ def test_parse_metadata_csv_non_ascii(self):
...
+        ]
+        with open('metadata.csv', 'wb') as f:
+            writer = csv.writer(f)
+            for row in data:
+                writer.writerow(row)
+
+        # Run test
+        dc = archivematicaCreateMETSMetadataCSV.parseMetadataCSV('metadata.csv')
+        # Verify
+        assert dc
```
Comment (mistydemeo) [inline src/MCPClient/tests/test_create_aip_mets.py:367]: Maybe test the number of rows?

Comment (Hwesta) [inline src/MCPClient/tests/test_create_aip_mets.py:367]: Added.

**Comment by mistydemeo** [comment]
:+1:

**Comment by Hwesta** [comment]
Also cherry-picked to 1.5 as 4de2099f7cf76c76ca45a58579722c4ef4f6c3f8

**Pull request merged**